final_prep/mod_zero_hero.rb

```diff
@@ -2,45 +2,93 @@
 
 # Declare two variables - hero_name AND special_ability - set to strings
 
+hero_name = 'The fly'
+special_ability = 'flying'
+
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
+greeting = 'hello, this is #{hero_name}'
+catchphrase = '#{special_ability} to the rescue!'
+
 # Declare two variables - power AND energy - set to integers
 
+power = 10
+energy = 10
+
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
 
+full_power = power * 500
+full_energy = energy + 150
+
 # Declare two variables - is_human and identity_concealed - assigned to booleans
 
+is_human = true
+identity_concealed = false
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
 
+arch_enemies = ['joe', 'bob', 'sue']
+sidekicks = ['jill', 'keith', 'stewart']
+
 # Print the first sidekick to your terminal
 
+print sidekicks[0]
+
 # Print the last arch_enemy to the terminal
 
+print arch_enemies[2]
```

Comment:
How would you print the last `arch_enemy` if you didn't know the length of the array?